toolkit/tools/pkg/imagecustomizerlib/partitionutils.go

```diff
@@ -293,43 +295,94 @@ func isSpecialPartition(fstabEntry diskutils.FstabEntry) bool {
 	}
 }
 
-func findSourcePartition(source string, partitions []diskutils.PartitionInfo) (string, error) {
-	_, partition, _, err := findSourcePartitionHelper(source, partitions)
+func findSourcePartition(source string, partitions []diskutils.PartitionInfo,
+	buildDir string,
+) (string, error) {
+	_, partition, _, err := findSourcePartitionHelper(source, partitions, buildDir)
 	if err != nil {
 		return "", err
 	}
 
 	return partition.Path, nil
 }
 
-func findSourcePartitionHelper(source string,
-	partitions []diskutils.PartitionInfo,
-) (imagecustomizerapi.MountIdentifierType, diskutils.PartitionInfo, int, error) {
-	mountIdType, mountId, err := parseSourcePartition(source)
+func findSourcePartitionHelper(source string, partitions []diskutils.PartitionInfo,
```

Comment:
`findSourcePartitionHelper` and `findExtendedSourcePartitionHelper` are the same function.